### PR TITLE
feat(PRL-5369, PRL-5659): Fix sonar issues

### PR DESCRIPTION
**Pull request by sealDebolina**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the enable_keep_helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/check-your-answers/content.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { HTML } from './common/htmlSelectors';
 import { ANYTYPE } from './common/index';
 import {
   ApplicantDetails,
-  CaseName,
+  //CaseName,
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/content.ts:18]: Remove CC

File: src/main/steps/c100-rebuild/check-your-answers/content.ts [hunk 2]
```diff
@@ -451,7 +451,7 @@ const safteyConcenFilledSection = (userCase, contentLanguage, language) => {
 export const commonSectionsForContentLoader = (contentLanguage, userCase, language) => {
   return {
     PostCodeAndTypeOfApplication: [
-      CaseName(contentLanguage, userCase),
+      //CaseName(contentLanguage, userCase),
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/content.ts:454]: Remove CC

File: src/main/steps/c100-rebuild/check-your-answers/mainUtil.test.ts [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { PartyType, YesOrNo } from '../../../app/case/definition';
 import { ANYTYPE } from './common/index';
 import {
   ApplicantDetails,
-  CaseName,
+  //CaseName,
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/mainUtil.test.ts:8]: Remove CC

File: src/main/steps/c100-rebuild/check-your-answers/mainUtil.test.ts [hunk 4]
```diff
@@ -1683,16 +1683,16 @@ describe('test cases for main util', () => {
...
-    const userCase = {
-      id: 'id',
-      state: undefined,
-      applicantCaseName: 'test',
-    } as ANYTYPE;
-    const CaseName_fun = CaseName({ sectionTitles, keys, Yes: 'Yes', No: 'No', content }, userCase);
-    expect(CaseName_fun?.rows).not.toBe([]);
-    expect(CaseName_fun?.title).toBe(undefined);
-  });
+  // test('CaseName - util', () => {
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/mainUtil.test.ts:1686]: Remove CC

File: src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts [hunk 5]
```diff
@@ -25,26 +25,32 @@ import {
...
-      value: userCase['applicantCaseName'],
-      changeUrl: Urls['C100_CASE_NAME'],
-    },
-  ];
-  return {
-    title: sectionTitles['caseName'],
-    rows: getSectionSummaryList(SummaryData, content),
-  };
-};
+// export const CaseName = (
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts:38]: Remove CC

File: src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts [hunk 6]
```diff
@@ -396,27 +383,27 @@ export const OtherChildrenDetails = (
...
-          }
-        );
-      } else {
-        newChildDataStorage.push({
-          key: keys['dobLabel'],
-          value: DATE_FORMATTOR(personalDetails['dateOfBirth'], language),
-          changeUrl: applyParms(Urls['C100_CHILDERN_OTHER_CHILDREN_PERSONAL_DETAILS'], { childId: id }),
-        });
-      }
+      populateDateOfBirth(personalDetails, newChildDataStorage, keys, language, id, false);
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts:386]: Remove CC

File: src/main/steps/common/documents/upload/utils.ts [hunk 7]
```diff
@@ -50,7 +50,7 @@ export const deleteDocument = async (req: AppRequest, res: Response): Promise<vo
   try {
     await client.deleteCitizenStatementDocument(query.documentId as string);
 
-    if (req.session.userCase && req.session.userCase.hasOwnProperty(uploadedFilesDataReference)) {
+    if (req.session.userCase.hasOwnProperty(uploadedFilesDataReference)) {
```
Comment (vivek-sekhar) [inline src/main/steps/common/documents/upload/utils.ts:53]: req.session?.userCase?.uploadedFilesDataReference

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/common/documents/upload/utils.ts (hunk 7 above)
Comment (sealDebolina) [reply src/main/steps/common/documents/upload/utils.ts:53]: Hi @vivek-sekhar , we cannot remove hasOwnProperty, as uploadedFilesDataReference is a constant which hold applicantUploadFiles' or 'respondentUploadFiles' depending on party type.

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**